Артём Швецов

shvetsovart opened the pull request:
https://task4-cyan.vercel.app/

shvetsovart committed:
add first version

shvetsovart committed:
added features to personality progress

shvetsovart committed:
added colors and percents to personality info

shvetsovart committed:
little fix of personality progress element

shvetsovart committed:
making everything look good on mobile devices

mokhov commented:
Так, ну в целом результат очень классный, к Accessibility почти не придраться, но все-таки.
1. Идея класть Button в ссылку приводит к двум фокусам кнопки, т.е при переборе через VoiceOver это поведение сохранится. Можно просто попробовать табом поперемещаться и увидишь это. Тут есть два вида решения, придумай на твоё усмотрение.
2. Скидывал скриншот как ведёт себя `border-radius: 50%` в Firefox — это тоже хорошо бы поправить.
3. Там где Skill proficiency percentage лучше использовать `input type=range` 
4. На мобилках `float: right` на мелких экранах в тех же скиллах выглядит не очень ок

shvetsovart committed:
fixed everything

buttons 2x focus, input range, float: right for small screens, border: 50% for firefox

shvetsovart commented:
Для бегунка в Add skill: увеличить высоту, добавить вывод процентов
В Traits убрать правый текст вниз

shvetsovart committed:
fixed range input, main trait label

shvetsovart committed:
Rename template.html to index.html

shvetsovart committed:
traits little screens fix

shvetsovart committed:
fixed skill form width issues

mokhov commented:
Принято
